### PR TITLE
[new-release] melange (3.0.0)

### PR DESCRIPTION
**Pull request by anmonteiro**
Toolchain to produce JS from Reason/OCaml

Project page: https://github.com/melange-re/melange

This release includes a version for each of the following OCaml compiler versions: 4.14 and 5.1.

Starting with 3.0, we're experimenting with releasing a version of Melange that works across more than one OCaml version, following the same release naming as Merlin.

### CODE REVIEW

**Comment by anmonteiro** [comment]
opened https://github.com/ocaml/opam-repository/pull/25141 to fix the revdeps issues.

File: packages/melange/melange.3.0.0-414/opam [hunk 1]
```diff
@@ -0,0 +1,48 @@
+version: "3.0.0-414"
```
Comment (mseri) [inline packages/melange/melange.3.0.0-414/opam:1]:
```suggestion
```

File: packages/melange/melange.3.0.0-51/opam [hunk 2]
```diff
@@ -0,0 +1,48 @@
+version: "3.0.0-51"
```
Comment (mseri) [inline packages/melange/melange.3.0.0-51/opam:1]:
```suggestion
```

**Comment by mseri** [comment]
Thanks!

**Pull request merged**